### PR TITLE
Site Settings Menu

### PR DESCRIPTION
**Pull request by hammeiam**
Create a menu popover for network, language, and theme settings. Hide the network UI behind a flag (currently disengaged). Network switching logic works so please test it locally by enabling the l2 flag at the bottom of the constants file.  

Screen Shot 2021-11-12 at 12 25 07 AM
Screen Shot 2021-11-12 at 12 24 52 AM

### CODE REVIEW

File: src/components/SiteSettingsMenu.module.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+.divider {
+  border: 1px solid var(--emphasis-2);
+  height: 1px;
+  width: 100%;
+  margin: 8px;
+}
+
+.section {
+}
```
Comment (ug02fast) [inline src/components/SiteSettingsMenu.module.scss:22]:
```suggestion
```
Is this being used as a placeholder?

**Review by ug02fast — APPROVED** [review]
Looks good. Approved but there might be some dead code lingering.

File: src/components/TopMenu.tsx [hunk 2]
```diff
@@ -54,12 +56,70 @@ function TopMenu({ activeTab }: Props): ReactElement {
...
+  return (
+    <div style={{ position: "relative" }} ref={wrapperRef}>
+      <Button
+        kind="ternary"
+        size="medium"
+        onClick={() => setIsDropdownOpen((state) => !state)}
+      >
+        <svg
+          width="6"
+          height="22"
```
Comment (sandrasong) [inline src/components/TopMenu.tsx:106]: Could you change this hamburger menu icon size to be 32 x 32 px?

**Review by sandrasong — COMMENTED** [review]
Looks great!

**Pull request merged**